--- /tmp/b-MainScaffold.kt	2026-07-04 03:53:58.961041764 +0000
+++ app/src/main/java/com/hashmm/app/ui/MainScaffold.kt	2026-07-04 03:51:38.782597843 +0000
@@ -20,10 +20,13 @@
 import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.filled.ChatBubble
 import androidx.compose.material.icons.outlined.Bolt
+import androidx.compose.material.icons.filled.Bolt
 import androidx.compose.material.icons.outlined.ChatBubbleOutline
 import androidx.compose.material.icons.outlined.Dashboard
+import androidx.compose.material.icons.filled.Dashboard
 import androidx.compose.material.icons.outlined.Lock
 import androidx.compose.material.icons.outlined.Person
+import androidx.compose.material.icons.filled.Person
 import androidx.compose.material3.Button
 import androidx.compose.material3.Surface
 import androidx.compose.material3.Icon
