--- /tmp/b-RemoteControlScreen.kt	2026-07-04 03:22:07.874791835 +0000
+++ app/src/main/java/com/hashmm/app/ui/remote/RemoteControlScreen.kt	2026-07-04 03:19:06.828764627 +0000
@@ -49,7 +49,6 @@
 import androidx.compose.material.icons.outlined.Close
 import androidx.compose.material.icons.outlined.Computer
 import androidx.compose.material.icons.outlined.ContentPaste
-import androidx.compose.material.icons.outlined.GridView
 import androidx.compose.material.icons.outlined.Keyboard
 import androidx.compose.material.icons.outlined.KeyboardArrowDown
 import androidx.compose.material.icons.outlined.KeyboardArrowUp
